Makefile

```diff
@@ -11,11 +11,11 @@ export OPERATOR_EXEC?=oc
 BUILD_GOPATH=$(TARGET_DIR):$(TARGET_DIR)/vendor:$(CURPATH)/cmd
 IMAGE_BUILDER?=docker
 IMAGE_BUILD_OPTS?=
-DOCKERFILE?=Dockerfile
+DOCKERFILE?=Dockerfile.webhook
```

Comment:
is this related to the PR ?

Reply:
sorry it was a mistake on my side, I'll drop that